teamA/jeongHaeMan1/routes/index.js

```diff
@@ -0,0 +1,9 @@
+//routes/index.js
+
+const express = require('express');
+const router = express.Router();
+
+const userRouter = require("./userRouter");
+router.use("/users", userRouter.router);
```

Comment:
코드의 가독성을 위해 외부 모듈을 불러오는 부분과 모듈의 속성을 지정해주는 부분을 분리하여 작성해주면 좋을 것 같습니다!
```suggestion
const userRouter = require("./userRouter");

router.use("/users", userRouter.router);
```